app/services/BulletPointBuilder.scala

```diff
@@ -330,11 +332,52 @@ object BulletPointBuilder {
   }
 
   def matchPhrases(p1: Phrase, p2: Phrase): Boolean = {
-    // Apply matching logic to English text as this is how Ocelot works currently
-    val (p1NoOfWordsToDisplay, p2NoOfWordsToDisplay, matchedWords) = matchInstructionText(p1.english, p2.english)
 
-    // Matching instructions must have matching leading text followed dissimilar trailing text
-    matchedWords.size >= matchLimit && (matchedWords.size < p1NoOfWordsToDisplay) && (matchedWords.size < p2NoOfWordsToDisplay)
+    if(useExplicitMatch(p1, p2)) {
```

Comment:
Both implicitMatchPhrases() and explicitMatchPhrases() could be removed and their single line content be substituted for their sole usage in matchPhrases(). Would reduce code and make the codes intention clearer

Reply:
Done.